### PR TITLE
Fixes cart item spacing

### LINKED ISSUE #8: Fix spacing in cart
Remove the two `<p>` elements for the start and end booking times and just use a single `<p>` element with a break element `<br/>`.

Some themes seem to hide break elements in the cart by setting the `display` to `none`, so this should be overridden.

### PR DESCRIPTION
**Pull request by mecha**
Resolves #8

### CODE REVIEW

File: templates/cart-booking-info.html [hunk 1]
```diff
@@ -1,4 +1,7 @@
 <div class="eddbk-cart-item-booking-info">
-    <p><b>${from_label}</b> <time datetime="${start_datetime}">${start_text}</time></p>
-    <p><b>${until_label}</b> <time datetime="${end_datetime}">${end_text}</time></p>
+    <p>
+        <b>${from_label}</b> <time datetime="${start_datetime}">${start_text}</time>
+        <br style="display: block !important;"/>
```
Comment (XedinUnknown) [inline templates/cart-booking-info.html]: The [`<br />`][br] is an inline element, not block.

[br]: https://www.w3.org/TR/html5/textlevel-semantics.html#the-br-element

**Review by XedinUnknown — CHANGES_REQUESTED** [review]

File: templates/cart-booking-info.html (hunk 1 above)
Comment (mecha) [reply templates/cart-booking-info.html]: Really? Then what makes it "break" the line?

Comment (XedinUnknown) [reply templates/cart-booking-info.html]: The fact that it represents a linebreak.

Comment (mecha) [reply templates/cart-booking-info.html]: Hah! No no, browsers apply a default style to it. One of those styles is what makes it break the line. Without those styles it's just an XML node.

@XedinUnknown So I did some research of my own and couldn't pinpoint what makes it break the line. But the problem is that some themes, for some reason, apply `display: none` to it, and applying `display: inline` seems to fix that. So I made the change anyway.

Comment (XedinUnknown) [reply templates/cart-booking-info.html]: What makes it break the line is the fact that it represents a linebreak, in the same way as the symbol "a" represents the letter "a" and that's why it gets rendered like that. That's all there is to it.

**Review by XedinUnknown — APPROVED** [review]

**Pull request merged**